lib/cli/index.js

```diff
@@ -77,6 +65,8 @@ function configureLogger(env) {
 
 // Options: Array cliArgs, Stream inputStream, Stream outputStream
 module.exports = function(options) {
+  willInterruptProcess.capture(options.process || process);
```

Comment:
`tests/helpers/ember` [requires ](https://github.com/ember-cli/ember-cli/pull/7086/files#diff-046dd933d1aae8e9131812190820a4c6R5) `cli/index`. If we capture the process in the top of the `cli/index` we will make it once for all tests. 

I think we need to capture the `process` before `cli` command started and [free it](https://github.com/ember-cli/ember-cli/pull/7086/files#diff-3386b9e4d97ad97b803f4678beb66c5aR143) after it's finished.